src/features/courses/course-controller.ts

```diff
@@ -3271,6 +3272,63 @@ class CourseController {
             return { problemScores: problemScoresReturn, bestVersionScore: bestVersionScoreReturn, bestOverallVersion: bestOverallVersionReturn};
         });
     };
+    async canUserGradeAssessment({
+        user,
+        topicId
+    }: CanUserGradeAssessmentOptions): Promise<boolean> {
+        const topic = await courseRepository.getCourseTopic({ id: topicId });
```

Comment:
Should this be 3 separate calls instead of one call with includes? (What is this, GraphQL?)

Reply:
I like it as 3 separate in some regards, I think it makes it easier to pass it in, but I also like it in one because it's potentially more optimized on it's own, I'm back and forth and I think in our stabilization sprint we will need to investigate the performance differences